recipes/mad/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "mad" %}
+{% set version = "0.15.1b" %}
+{% set sha256 = "bbfac3ed6bfbc2823d3775ebb931087371e142bb0e9bb1bee51a76a6e0078690" %}
+
+package:
+  name: lib{{ name|lower }}
```

Comment:
The name of the package is `libmad` however the directory the recipe is in is `mad`. Could you please send a follow-up PR to staged-recipes to fix this discrepancy at @dschreij ? Currently this recipe (and a few others) are stuck in limbo until that happens.